roles/tests-unit/files/FWO.Test/HtmlToPdfTest.cs

```diff
@@ -16,7 +15,7 @@ internal class HtmlToPdfTest
     {
         private const string FilePath = "pdffile.pdf";
         private const string Html = "<html><body><h1>test</h1><h2>test mit puppteer</h2></body></html>";
-        private const string ChromeBinPathLinux = "/usr/local/bin";
+        private const string ChromeBinPathLinux = "/usr/local/fworch/bin";
```

Comment:
could we move all const strings into FWO.Basics constants and thereby also avoid duplicates?